src/apps/statements/expressPresenter/utils/validateStatementBypassQueues.ts

```diff
@@ -0,0 +1,13 @@
+import { createWarning, Warnings } from 'rulr';
+
+export const validateStatementBypassQueues = (
+  statementBypassQueues: string[] | undefined,
```

Comment:
@lucuilearningpool , if you're validating `req.query.bypassQueues` which is `undefined` or `string` it means this function will behave in a wrong way